libraries/chain/hotstuff/test/hotstuff_tools.cpp

```diff
@@ -1,63 +1,14 @@
-#include <boost/test/unit_test.hpp>
+#define BOOST_TEST_MODULE hotstuff
+
```

Comment:
`hotstuff_tools.cpp` seems like an odd name for this file. Maybe rename to `hotstuff_misc_tests.cpp`

Reply:
Changed to `finality_misc_tests.cpp` for eventual renaming `hotstuff` directory to `finality` directory.